src/views/CoCurricularTranscript/index.js

```diff
@@ -348,6 +357,15 @@ export default class Transcript extends Component {
           </Card>
         </div>
       );
+    } else if (this.userIsFacStaff()) {
+      return (
+        <GordonLimitedAvailability
+          pageName="Experience Transcript"
+          backToLocation="My Profile"
+          backToLink="/myprofile"
+          availableTo="students and alumni"
+        />
+      );
```

Comment:
Technically, you're still checking this twice, since you invoke the function twice. What I meant was move this return path above the other one, so you check if the user is facstaff and, if so, show them the Unavailable page, otherwise show them the Experience Transcript.

Reply:
Ah, yes. That makes sense!

Reply:
Done.